cinema/migrations/0004_alter_movie_image.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.1 on 2024-10-16 16:47
+
+import cinema.models
```

Comment:
The import statement `import cinema.models` is not used in this migration file and can be removed to clean up the code.